modules/install-acs-operator.adoc

```diff
@@ -42,4 +42,4 @@ If you choose manual updates, you must update the {product-title-short} Operator
 * After the installation completes, navigate to *Operators* -> *Installed Operators* to verify that the {product-title} Operator is listed with the status of *Succeeded*.
 
 .Next Step
-* Install, configure, and deploy the `Central` custom resource.
+* You've installed the Operator into the *rhacs-operator* project. Now, using that Operator, you install, configure, and deploy the `Central` custom resource.
```

Comment:
I'm not sure you need the introductory sentence you added. It seems obvious to me. If you want to leave it:

```suggestion
* You installed the Operator into the *rhacs-operator* project. Using that Operator, you install, configure, and deploy the `Central` custom resource.
```

Reply:
People seem to be having confusion about installing the Operator (goes into the `rhacs-operator` namespace) and then using the Operator to install Central and Secured Cluster services (goes into the `stackrox` namespace). I'd like to keep something there to alleviate that (hopefully).